guidelines/enhancement_template.md

```diff
@@ -115,6 +117,23 @@ bogged down.
 
 Include a story on how this proposal will be operationalized:  lifecycled, monitored and remediated at scale.
 
+### API Extensions
+
+API Extensions are CRDs, admission and conversion webhooks, aggregated API servers,
+finalizers, i.e. those mechanisms that change the OCP API surface and behaviour.
```

Comment:
CRDs certainly have less operation impact. But listing them is a good idea IMO.

I fear a little that we have to try to keep the balance. We can add more and more to stop certain common mistakes from happening. But the risk is that ppl neglect certain sections because there are just too many. I notice this a lot with the risk, test and version skew section at the bottom. Often they are not seriously filled in.

Reply:
can we give more guidance about when those sections are applicable?  my fear is that someone adding a new CRD is going to have no idea what to put in the "SLI" section (i don't really know what we'd expect them to put there)